java/interview/largest-killstreak.java

```diff
@@ -31,4 +55,40 @@
  * 
  * This is more commonly the actual definition of a multi killstreak found in games.
  */
-public int[][] largestKillstreak(int[][] kills) { }
\ No newline at end of file
+public List<Integer> largestKillstreak(int[][] kills) {
+  Arrays.sort(kills, (a, b) -> a[0] - b[0]);
+  // Mapping of userId -> [ killCount, killTime ].
+  HashMap<Integer, int[]> map = new HashMap<Integer, int[]>();
+
+  int max = Integer.MIN_VALUE;
+  for (int i = 0; i < kills.length; i ++) {
+    int timestamp = kills[i][0];
+    int killer = kills[i][1];
+    int victim = kils[i][2];
+
+    // Reset the victim's kill count.
```

Comment:
hmm so im not sure this should reset the kill count for all victims. The edge case im thinking is that the victim has the highest killcount before dying, but if we reset here, then we won't be able to find it later when creating the resultant list.